charts/datadog/values.yaml

```diff
@@ -2344,6 +2344,12 @@ providers:
     # providers.gke.autopilot -- Enables Datadog Agent deployment on GKE Autopilot
     autopilot: false
 
+    # providers.gke.autopilotAllowlists -- Specify the Datadog WorkloadAllowlists to be synchronized on GKE Autopilot.
+    # If left empty, then the default latest WorkloadAllowlist will be used. To synchronize all available allowlists,
+    # set to "[Datadog/datadog/*]".
+    autopilotAllowlists: []
+    #  - Datadog/datadog/*
+
```

Comment:
I'm considering removing this option since we only have 1 Datadog WorkloadAllowlist and exposing the option to specify allowlist versions could introduce potential for user-error.